bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -337,6 +372,13 @@
 		<category>BatteryLevel</category>
 		<state pattern="%.0f %unit%" readOnly="true"/>
 	</channel-type>
+	<channel-type id="vehicleTemperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
```

Comment:
Did you consider to make it a bit more specific e.g. 'Inside Temperature' or 'Battery Temperature' or similar?

Reply:
Could be water (boiler), room temperature or ...
Depend on the actual use case. Evcc is a very versatile tool.